scss/6-components/_callout.scss

```diff
@@ -11,24 +11,25 @@
 
     .cads-callout-label {
         display: none;
-        position: absolute;
-        padding: 0.75em;
-        top: -1em;
-        left: -$cads-border-width-large;
-        font-weight: $cads-font-weight__bold;
+        position: relative;
         font-size: 0.75em;
-        line-height: 0.75em;
-        color: $cads-palette__white;
+        color: $cads-font-colour__base;
+        margin-bottom: 1em;
```

Comment:
Switch to spacing var?

Reply:
Yes that would be great, also seeing online now I think the spacing might be too large.